lib/datadog/core/configuration/option.rb

```diff
@@ -235,8 +232,10 @@ def validate(type, value)
           case type
           when :string
             value.is_a?(String)
-          when :int, :float
-            value.is_a?(Numeric)
+          when :int
+            value.is_a?(Integer)
+          when :float
+            value.is_a?(Float) || value.is_a?(Rational)
```

Comment:
I think allowing users to provide integer values for floats is a nice UX touch -- e.g. if you're providing a percentage, often you may want to say 50 and not 50.0 or something like that.

So I don't think we should lose that.